src/components/Table/ListTable/index.vue

```diff
@@ -124,6 +124,7 @@ export default {
       const has =
         (this.quickFilters && this.quickFilters.length > 0) ||
         (this.quickSummary && this.quickSummary.length > 0)
+
       return !!has
     },
     dataTable() {
```

Comment:
There is no difference between the two versions provided. The code appears to be identical, with only one line added at the beginning of `dataTable()`. No issues were found during the review.